packages/xstate-wallet/src/workflows/application.ts

```diff
@@ -121,6 +122,7 @@ const generateConfig = (
   actions: WorkflowActions,
   guards: WorkflowGuards
 ): MachineConfig<WorkflowContext, WorkflowStateSchema, WorkflowEvent> => ({
+  strict: !IS_PRODUCTION,
```

Comment:
Why?

Reply:
I was attempting to be able to log the store dump only on error: https://github.com/statechannels/monorepo/pull/1491/commits/8b7d3df4b01072235eb7b918e1ddcbf57de20ed1

Eventually, I decided to log the store on every transition, because it doesn't not really pollute the logs.

Perhaps it should just be set to `strict: true`, but I am not sure if we want that.